tests/phpunit-compare.xml.dist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is the same as the "integration" one from #1581 no?

Needs to be removed from this PR?

Reply:
file removed